File: C_Vlad_and_a_Sum_of_Sum_of_Digits.java
import java.util.Scanner;

public class C_Vlad_and_a_Sum_of_Sum_of_Digits {
    
    public static void main(String[] args) {
       Scanner input = new Scanner(System.in);
       int testCase = input.nextInt();
       for (int i = 0; i < testCase; i++) {
            int num = input.nextInt();
            totalSum(num);
       }
       input.close();
    }

    public static void totalSum(int num){
        Long[] prefix = new Long[(int)(2 * (Math.pow(10, 5) + 1))];
        prefix[0] = 0L;
        for(int i = 1; i <= 2 * (Math.pow(10, 5)); i++){
            prefix[i] = prefix[i - 1] + sumOfDigits(i);
        }
        System.out.println(prefix[num]);
    }

    public static int sumOfDigits(long number) {
        String numberAsString = Long.toString(number);
        int sum = 0;
        for (int i = 0; i < numberAsString.length(); i++) {
            sum += Character.getNumericValue(numberAsString.charAt(i));
        }
        return sum;
    }
}

File: C_Vlad_and_a_Sum_of_Sum_of_Digits.py

prerfix = [0] * (2 * (10 ** 5) + 1)
for i in range(1, 2 * (10 ** 5) + 1):
    prerfix[i] = sum(map(int, list(str(i)))) + prerfix[i-1]


tesst = int(input())
for i in range(tesst):
    num = int(input())
    print(prerfix[num])